packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -134,8 +140,24 @@ const SigninContainer = ({
   const sensitiveDataClient = useSensitiveDataClient();
   const shouldUseReactEmailFirst = useCheckReactEmailFirst();
 
-  const { queryParamModel, validationError } =
-    useValidatedQueryParams(SigninQueryParams);
+  const { queryParamModel, validationError } = useValidatedQueryParams(
+    SigninQueryParams,
+    true
+  );
+
+  // Validates that query parameters are valid for an oauth integration
+  useValidatedQueryParams(
```

Comment:
Testing locally - if I introduce errors on oAuth params I see a blank page instead of the AppErrorBoundary

Reply:
Let me double check this.

Reply:
Re-tested and this is working now! Thanks for the fix.